sw/sw.mk

```diff
@@ -63,7 +63,20 @@ CHS_SW_FLAGS := ${CHS_SW_32_FLAGS}
 endif
 
 CHS_SW_DIR       ?= $(CHS_ROOT)/sw
-CHS_SW_LD_DIR    ?= $(CHS_SW_DIR)/link
+
+CHS_MEMISL ?= 0
+
+ifeq (${CHS_MEMISL}, 0)
+
+CHS_SW_LD_DIR    ?= $(CHS_SW_DIR)/link/cheshire
+
+endif
+ifeq (${CHS_MEMISL}, 1)
+
+CHS_SW_LD_DIR    ?= $(CHS_SW_DIR)/link/chimera
+
+endif
```

Comment:
I would replace this with a simple and beautiful 
`CHS_SW_LD_DIR    ?= $(CHS_SW_DIR)/link/cheshire`
That way you can override it upstream (in chimera, or any other project), and cheshire remains unaware of chimera's existence.

Reply:
Make sense to me